docs/topic_guides/evaluation/edge_metrics.md

```diff
@@ -0,0 +1,198 @@
+
+$$\textsf{Specificity} = \frac{\textsf{True Negatives}}{\textsf{All Negatives}} = \frac{\textsf{True Negatives}}{\textsf{True Negatives} + \textsf{False Positives}}$$
+
+??? info "Specificity in Splink:"
+
+    - [accuracy_chart_from_labels_table](../../charts/accuracy_chart_from_labels_table.ipynb)
+    - [truth_space_table_from_labels_column](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_column)
+    - [truth_space_table_from_labels_table](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_table)
+
+
+### Positive Predictive Value (Precision)
+
+The Positive Predictive Value (Precision), is the proportion of predicted matches which are true matches.
+
+$$\textsf{Precision} = \frac{\textsf{True Positives}}{\textsf{All Predicted Positives}} = \frac{\textsf{True Positives}}{\textsf{True Positives} + \textsf{False Negatives}}$$
+
+??? info "Precision in Splink:"
+
+    - [accuracy_chart_from_labels_table](../../charts/accuracy_chart_from_labels_table.ipynb)
+    - [truth_space_table_from_labels_column](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_column)
+    - [truth_space_table_from_labels_table](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_table)
+    - [precision_recall_chart_from_labels_table](../../charts/precision_recall_chart_from_labels_table.ipynb)
+
+### Negative Predictive Value
+
+The Negative Predictive Value is the proportion of predicted non-matches which are true non-matches.
+
+$$\textsf{Negative Predictive Value} = \frac{\textsf{True Negatives}}{\textsf{All Predicted Negatives}} = \frac{\textsf{True Negatives}}{\textsf{True Negatives} + \textsf{False Positives}}$$
+
+??? info "Negative Predicitive Value in Splink:"
+
+    - [accuracy_chart_from_labels_table](../../charts/accuracy_chart_from_labels_table.ipynb)
+    - [truth_space_table_from_labels_column](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_column)
+    - [truth_space_table_from_labels_table](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_table)
+
+
+!!! warning 
+
+    Each of these metrics looks at just one row or column of the confusion matrix. A model cannot be meaningfully summarised by just one of these performance measures.
+
+    **“Predicts cancer with 100% Precision”** - is true of a “model” that correctly identifies one known cancer patient, but misdiagnoses everyone else as cancer-free.
+
+    **“AI judge’s verdicts have Recall of 100%”** - is true for a power-mad AI judge that declares everyone guilty, which would be stoically described by civil servants as “sub-optimal”.
```

Comment:
This gave me a good chuckle

Reply:
Ah, I meant to change this actually. This was copied over from Sam's original confluence article haha

Reply:
I have lightened the language in this slightly for public consumption